enterprise/frontend/src/embedding-sdk/package.template.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@metabase/embedding-sdk-react",
-  "version": "0.0.3",
-  "description": "Metabase Embedding SDK",
+  "name": "metabase-embedding-sdk-react",
+  "version": "0.1.1",
```

Comment:
needed because I published 0.1.0, but that version did not include a README.md in the output, so I published 0.1.1 with the README included.